gulpfile.js

```diff
@@ -131,6 +131,20 @@ function runTest(name, callback) {
 			};
 			fs.mkdirSync(output);
 			test(newTS, lib[1], output, reporter).on('finish', function() {
+				if (path.sep === '\\') { //filenames embedded in error output contain OS-dependent path separators
+					errors.forEach(function(err) {
+						var colon = err.message.indexOf(":");
+						if (colon === -1 || !err.diagnostic || err.message.indexOf(path.sep) === -1) {
+							return;
+						}
+
+						var fileName = err.message.slice(0, colon);
+						var detail = err.message.slice(colon);
+						fileName = fileName.split(path.sep).join('/');
```

Comment:
Use fileName.replace(...) with a regex. (A regex can match multiple occurrences, a string would only match the first).